core/unit_test/TestCTestDevice.cpp

```diff
@@ -76,6 +66,8 @@ void ctest_environment::SetUp() {
   setenv("CTEST_RESOURCE_GROUP_9_GPUS", "id:4,slots:1", 1);
 }
 
+struct ctest_environment_DeathTest : public ctest_environment {};
```

Comment:
Why do you create a derived class?

Reply:
We have that convention that all death tests must be categorized with a tag ending in `_DeathTest`.
When using `TEST_F`, the name is the test fixture class so I need a strongly typed class that does the same thing as the basic fixture.